examples/visualization/plot_evoked_topomap_delayed_ssp.py

```diff
@@ -38,7 +38,9 @@
 # delete EEG projections (we know it's the last one)
 raw.del_proj(-1)
 # add ECG projs for magnetometers
-[raw.add_proj(p) for p in mne.read_proj(ecg_fname) if 'axial' in p['desc']]
+for p in mne.read_proj(ecg_fname):
+    if 'axial' in p['desc']:
+        raw.add_proj(p)
```

Comment:
here I'm really not sure we need to follow thei advice. List comprehehsions are often used as expressions without assignment, even outside of function calls. I would accept this for module code but in an example there is no risk by not assigning an expressions output. To me the benefits of keeping it are higher than the codacy issue.